torchvision/prototype/transforms/functional/_color.py

```diff
@@ -19,6 +23,10 @@ def adjust_brightness(inpt: features.InputTypeJIT, brightness_factor: float) ->
 adjust_saturation_image_pil = _FP.adjust_saturation
 
 
+def adjust_saturation_video(video: torch.Tensor, saturation_factor: float) -> torch.Tensor:
```

Comment:
Following the pattern with other kernels, shouldn't this be something like:
```suggestion
def adjust_saturation_video(video: features.VideoTypeJIT, saturation_factor: float) -> features.VideoTypeJIT:
```

Reply:
No. The kernels take plain tensors and return them. As such annotating with `torch.Tensor` is correct as this is not a workaround for JIT, but what we actually want.